controllers/suite_test.go

```diff
@@ -172,6 +175,11 @@ var _ = BeforeSuite(func() {
 		})
 		Expect(err).ToNot(HaveOccurred())
 
+		crdList := map[string]bool{}
+		for _, CRDNames := range GetCRDNames() {
+			crdList[CRDNames] = true
+		}
```

Comment:
- I'm tending towards to believe the motive for moving from earlier design of using an enum is to enable tests?
- Or there any other reason?

Reply:
Yes, it will also enable us to deploy the resources based on whether or not CRD exists.